### DISCUSSION TITLE
Move the cluster and devices

### DISCUSSION THREAD

**Discussion by SerhiiZahuba** [netbox-community/netbox#10261] [General]
I need to move the cluster and the devices in it to another site.

If I try to move the device to another site then the error is "The assigned cluster belongs to a different site"

If I try to move the cluster the error is " devices are assigned as hosts for this cluster but are not in site ". How to make this move?

**Comment by candlerb** [comment]
Clusters don't have to be assigned to any site.  So, first set the cluster site to null; then move the devices; then you can set the cluster site to the desired value.

**Comment by SerhiiZahuba** [comment]
Thanks
